### ISSUE TITLE
Author on posts

### ISSUE THREAD

**Issue by stefanolsenn** [PiranhaCMS/piranha.core#610]
Hi @tidyui 

What do you think about an author attribute on posts, so the user can see who created the post? If it's fine by you, I can submit a PR when I'm done with it.

**Comment by tidyui** [comment]
Yes we’ve been discussing this for a while. As manager authentication is decoupled from Piranha we can’t rely on author information to be present on the user entity. For this reason we would need a separate data entity for just authors, and maybe an associated table that connects a user id to an author so a user can have a default author. Then a Dropdown on Posts (and maybe even pages) where you can connect the author.

**Comment by stefanolsenn** [comment]
Cool, sounds like an okay solution to it. I'll let you know when I have something :)

**Comment by CrossBound** [comment]
I am interested in this feature. Any update for this?

**Comment by tidyui** [comment]
I'm interested in it too, and now that `7.0` is released maybe we could take some time to investigate it further!

**Comment by CrossBound** [comment]
I'm confused by the comment above. Anytime a post is created there is **_always_** a user that created it... at the time of creation why could the current user not be saved with the post as meta-data? Why the need for the author table?

**Comment by tidyui** [comment]
Hi! The user/login functionality is totally abstracted from Piranha to enable integration with different auto providers. With this said there’s nothing to guarantee that the current user holds any relevant meta data. The Author table would simply be to standardize meta data in a single place and connect it to the current user.

**Comment by barcollin** [comment]
How far is this issue. I am interested in adding Author information that would be attached to each of my post. Are they any other detailed solutions on this case.

**Comment by lucasriechelmann** [comment]
Hi @tidyui 
I do not know if it would be the best way
Why do not put it like it is done with the Category?
The author could input his name there and could select it in the next post.
![author piranha](https://user-images.githubusercontent.com/10465626/213785229-0bf2dce7-d36a-40df-b144-92e702843e16.png)

**Comment by LuisLepe25** [comment]
> Yes we’ve been discussing this for a while. As manager authentication is decoupled from Piranha we can’t rely on author information to be present on the user entity. For this reason we would need a separate data entity for just authors, and maybe an associated table that connects a user id to an author so a user can have a default author. Then a Dropdown on Posts (and maybe even pages) where you can connect the author.

I understand this but it doesn't need linked to user, it will be trusted for any user who has the claims to create pages, if that user selects that he is author "John Wick" then he is.

This should be added as a table on piranha "Piranha users", on the manager we should be able to create authors with some information, then there should be  a block for author and select an author already loaded on the manager, then it can redirect to page of author.
